<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="2" hostname="DESKTOP-FVRAVKU" name="com.core.qtpselenium.ddf.testcases.suite.portfolio.LoginTest" tests="3" failures="1" timestamp="2021-11-24T08:18:43 IST" time="16.264" errors="0">
  <testcase name="doLogin" time="8.887" classname="com.core.qtpselenium.ddf.testcases.suite.portfolio.LoginTest">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Titles did not match for login page">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Titles did not match for login page
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at com.core.qtpselenium.ddf.keywords.UtilityKeywords.assertAll(UtilityKeywords.java:82)
at com.core.qtpselenium.ddf.testcases.suite.portfolio.LoginTest.doLogin(LoginTest.java:48)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- doLogin -->
  <testcase name="doLogin" time="0.290" classname="com.core.qtpselenium.ddf.testcases.suite.portfolio.LoginTest">
    <skipped/>
  </testcase> <!-- doLogin -->
  <testcase name="doLogin" time="7.087" classname="com.core.qtpselenium.ddf.testcases.suite.portfolio.LoginTest">
    <skipped/>
  </testcase> <!-- doLogin -->
</testsuite> <!-- com.core.qtpselenium.ddf.testcases.suite.portfolio.LoginTest -->
